### PR TITLE
sync lo address to the asic

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

**Comment by lguohan** [comment]
#103

File: orchagent/intfsorch.cpp [hunk 1]
```diff
@@ -64,6 +63,13 @@ void IntfsOrch::doTask(Consumer &consumer)
         string op = kfvOp(t);
         if (op == SET_COMMAND)
         {
+            if (alias == "lo")
+            {
+                addIp2MeRoute(ip_prefix);
+                it = consumer.m_toSync.erase(it);
+                continue;
+            }
+
```
Comment (qiluo-msft) [inline orchagent/intfsorch.cpp]: suggest move 'alias == lo' block to outside, because it is similar to eth0/docker0. Then 'getPort' can be also outside.

Comment (stcheng) [inline orchagent/intfsorch.cpp]: if this is moved to outside, then there will be if/else for the op == SET/DEL.

Comment (qiluo-msft) [inline orchagent/intfsorch.cpp]: Agree. it is clearer.

**Pull request merged**